src/applications/personalization/profile/components/contact-information/ContactInformation.jsx

```diff
@@ -28,21 +29,41 @@ const getScrollTarget = hash => {
   return document.querySelector(hashWithoutLeadingEdit);
 };
 
-const ContactInformation = ({
-  clearSuccessAlert,
-  hasUnsavedEdits,
-  hasVAPServiceError,
-  showDirectDepositBlockedError,
-  openEditModal,
-}) => {
+const ContactInformation = () => {
   const lastLocation = useLastLocation();
-  useEffect(() => {
-    document.title = `Contact Information | Veterans Affairs`;
 
-    return () => {
-      clearSuccessAlert();
-    };
-  }, []);
+  const showDirectDepositBlockedError = useSelector(
+    state => !!cnpDirectDepositIsBlocked(state),
+  );
+  const hasUnsavedEdits = useSelector(
+    state => state.vapService.hasUnsavedEdits,
+  );
+  const hasVAPServiceError = useSelector(state =>
+    hasVAPServiceConnectionError(state),
+  );
+
+  const directDepositIsAlwaysShowing = useSelector(
+    profileAlwaysShowDirectDepositDisplay,
+  );
+
+  const dispatch = useDispatch();
+  const clearSuccessAlert = useCallback(
+    () => dispatch(clearMostRecentlySavedField()),
+    [dispatch],
+  );
+
+  const openEditModal = useCallback(() => dispatch(openModal()), [dispatch]);
+
+  useEffect(
+    () => {
+      document.title = `Contact Information | Veterans Affairs`;
+
+      return () => {
+        clearSuccessAlert();
+      };
+    },
+    [clearSuccessAlert],
+  );
```

Comment:
I had a potential idea for this. It seems repeated and unnecessary to be setting this statically. The page name ('Contact Information' in this case) lives in the route declaration, so my thought was to do this somewhere like the ProfileWrapper and use those routes as the source of truth for the document title.

Obviously wouldn't be something to work into this PR, but @mdewey do you think this would be a good idea or over engineering?

Reply:
I think that's a good use of the wrapper components